panic: interface conversion: interface {} is []interface {}, not []string

When trying to execute `server:rescue` the following panic happens in a certain case. 

```
panic: interface conversion: interface {} is []interface {}, not []string

goroutine 1 [running]:
github.com/nl2go/hrobot-cli/cmd.(*RobotApp).NewServerActivateRescueCmd.func1(0xc0002a2280, 0x1924be8, 0x0, 0x0)
	/Users/development/hrobot-cli/cmd/server.go:227 +0x14a3
github.com/spf13/cobra.(*Command).execute(0xc0002a2280, 0x1924be8, 0x0, 0x0, 0xc0002a2280, 0x1924be8)
	/Users/development/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x2aa
github.com/spf13/cobra.(*Command).ExecuteC(0xc00028d400, 0xc0000ce930, 0xc00028d400, 0xc00022aba0)
	/Users/development/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb
github.com/spf13/cobra.(*Command).Execute(...)
	/Users/development/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
github.com/nl2go/hrobot-cli/cmd.(*RobotApp).Run(0xc00022aba0, 0xc00022aba0, 0x10)
	/Users/development/hrobot-cli/cmd/app.go:34 +0x6c
main.main()
	/Users/development/hrobot-cli/main.go:25 +0x1f4
```

--- a/README.md
+++ b/README.md
@@ -39,6 +39,13 @@ Run latest version from docker registry:
 
     docker run -it --env-file=sample.env registry.github.com/nl2go/hrobot-cli:v0-1-1
 
+## Update hrobot-go mocks
+
+This project uses `gomock`. When the hrobot-go library is upgraded to a new version, the mocks used 
+for the tests need to be updated. This can be done by running the following command:
+
+    mockgen -package mock github.com/nl2go/hrobot-go RobotClient > test/mock/mock_robot_client.go
+
 ## Features & overview
 
 Currently implemented commands are:

--- a/cmd/server.go
+++ b/cmd/server.go
@@ -224,12 +224,18 @@ func (app *RobotApp) NewServerActivateRescueCmd() *cobra.Command {
 				return
 			}
 
-			selectOs := rescueOptions.Os.([]string)
+			selectOsItems := rescueOptions.Os.([]interface{})
 
 			promptRescueOS := promptui.Select{
 				Label: "Select rescue operating system",
-				Items: selectOs,
+				Items: selectOsItems,
 				Size:  10,
+			}
+
+			// convert interface to string
+			selectOs := make([]string, len(selectOsItems))
+			for i, v := range selectOsItems {
+				selectOs[i] = v.(string)
 			}
 
 			chosenOSIdx, _, err := promptRescueOS.Run()

--- a/go.mod
+++ b/go.mod
@@ -10,7 +10,7 @@ require (
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/manifoldco/promptui v0.3.2
 	github.com/mattn/go-runewidth v0.0.6 // indirect
-	github.com/nl2go/hrobot-go v0.1.2
+	github.com/nl2go/hrobot-go v0.1.3
 	github.com/sirupsen/logrus v1.4.2
 	github.com/spf13/cobra v0.0.5
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15

--- a/test/mock/mock_robot_client.go
+++ b/test/mock/mock_robot_client.go
@@ -63,6 +63,36 @@ func (mr *MockRobotClientMockRecorder) BootRescueSet(arg0, arg1 interface{}) *go
 	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "BootRescueSet", reflect.TypeOf((*MockRobotClient)(nil).BootRescueSet), arg0, arg1)
 }
 
+// FailoverGet mocks base method
+func (m *MockRobotClient) FailoverGet(arg0 string) (*models.Failover, error) {
+	m.ctrl.T.Helper()
+	ret := m.ctrl.Call(m, "FailoverGet", arg0)
+	ret0, _ := ret[0].(*models.Failover)
+	ret1, _ := ret[1].(error)
+	return ret0, ret1
+}
+
+// FailoverGet indicates an expected call of FailoverGet
+func (mr *MockRobotClientMockRecorder) FailoverGet(arg0 interface{}) *gomock.Call {
+	mr.mock.ctrl.T.Helper()
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "FailoverGet", reflect.TypeOf((*MockRobotClient)(nil).FailoverGet), arg0)
+}
+
+// FailoverGetList mocks base method
+func (m *MockRobotClient) FailoverGetList() ([]models.Failover, error) {
+	m.ctrl.T.Helper()
+	ret := m.ctrl.Call(m, "FailoverGetList")
+	ret0, _ := ret[0].([]models.Failover)
+	ret1, _ := ret[1].(error)
+	return ret0, ret1
+}
+
+// FailoverGetList indicates an expected call of FailoverGetList
+func (mr *MockRobotClientMockRecorder) FailoverGetList() *gomock.Call {
+	mr.mock.ctrl.T.Helper()
+	return mr.mock.ctrl.RecordCallWithMethodType(mr.mock, "FailoverGetList", reflect.TypeOf((*MockRobotClient)(nil).FailoverGetList))
+}
+
 // GetVersion mocks base method
 func (m *MockRobotClient) GetVersion() string {
 	m.ctrl.T.Helper()

